test/mongo_util.py

```diff
@@ -81,6 +84,7 @@ def get_config() -> Tuple[MongoConfigTuple, dict[str, str]]:
         TEST_DATABASE_NAME: db_name,
         TEST_DATABASE_USER: mongo_user,
         TEST_DATABASE_PWD: mongo_pwd,
+        TEST_MONGO_RETRYWRITES: mongo_retrywrites == "true",
```

Comment:
```suggestion
        TEST_MONGO_RETRYWRITES: mongo_retrywrites,
```
The test code should provide the config as the code under test will receive it outside of tests

Reply:
👍

Reply:
:+1: